config/locales/devise.en.yml

```diff
@@ -32,10 +32,16 @@ en:
       success: "Successfully authenticated from %{kind} account."
     passwords:
       no_token: "You can't access this page without coming from a password reset email. If you do come from a password reset email, please make sure you used the full URL provided."
+      nonexisten_email: "Unable to find user with email '%{email}'."
```

Comment:
`non_existent_email`  -> You missed a `t` . 

Also, usually web applications shouldn't say when an email exists or not. Depending on what the user is trying to do, you may show a more generic message.